policy/interface.go

```diff
@@ -34,9 +34,9 @@ var ErrNotFound = errors.New("not found")
 // PDPBackend is the interface for the policy backend
 // Both the remote and local policy backend implement this interface
 type PDPBackend interface {
-	// PresentationDefinition returns the PresentationDefinition for the given scope
+	// PresentationDefinitions returns the PresentationDefinitions (as PEXPolicy) for the given scope
 	// scopes are space delimited. It's up to the backend to decide how to handle this
-	PresentationDefinition(ctx context.Context, authorizer did.DID, scope string) (*pe.PresentationDefinition, error)
+	PresentationDefinitions(ctx context.Context, authorizer did.DID, scope string) ([]pe.MultiPEX, error)
```

Comment:
Naming issues, but I don't have the final solution either. But:

- There's nothing "multi" about the `MultiPEX` struct? (it just wraps the PD in a struct with an audience attached).
- Would a `map[Audience]pe.PresentationDefinition` (or `map[Audience][]pe.PresentationDefinition`) work just as well? (instead of `[]pe.MultiPEX`)
-`Audience(Type)` is tricky in the context of OAuth2/JWTs, I understand it's "the audience type for the Presentation Definition", but it feels a bit too clever. "WalletType" would be too dumbed down?